src/components/Tooltip/Tooltip.tsx

```diff
@@ -236,43 +210,50 @@ export function Tooltip<FCProps = DefaultTooltipProps>(
             break
         }
       }
-    }, [isVisible])
+    }
+  }, [isVisible])
```

Comment:
I _think_ this `useEffect` dependency is responsible for a small jitter that can be seen on the first hover, since the position state is not set until the tool tip is visible. Is it possible to set the position during initial render?


https://user-images.githubusercontent.com/15805554/115771319-ccf75c00-a37b-11eb-879c-a29fc01d2826.mov

Reply:
It looks like USWDS gets around [this using a `setTimeout`](https://github.com/uswds/uswds/blob/v2.10.3/src/js/components/tooltip.js#L205), I updated it so that the `is-visible` class isn't turned on until _after_ the positioning values are updated (also using `useEffect`). let me know what you think!